v2v-helper/pkg/utils/utils.go

```diff
@@ -103,3 +107,56 @@ func WriteToLogFile(message string) error {
 	}
 	return nil
 }
+
+func atoi(s string) int {
+	i, err := strconv.Atoi(s)
+	if err != nil {
+		return 0
+	}
+	return i
+}
+
+func GetVjailbreakSettings(ctx context.Context, k8sClient client.Client) (*VjailbreakSettings, error) {
+	vjailbreakSettingsCM := &corev1.ConfigMap{}
+	if err := k8sClient.Get(ctx, k8stypes.NamespacedName{Name: constants.VjailbreakSettingsConfigMapName, Namespace: constants.NamespaceMigrationSystem}, vjailbreakSettingsCM); err != nil {
+		return nil, errors.Wrap(err, "failed to get vjailbreak settings configmap")
+	}
+
+	if vjailbreakSettingsCM.Data == nil {
+		return &VjailbreakSettings{
+			ChangedBlocksCopyIterationThreshold: constants.ChangedBlocksCopyIterationThreshold,
+			VMActiveWaitIntervalSeconds:         constants.VMActiveWaitIntervalSeconds,
+			VMActiveWaitRetryLimit:              constants.VMActiveWaitRetryLimit,
+			DefaultMigrationMethod:              constants.DefaultMigrationMethod,
+			VCenterScanConcurrencyLimit:         constants.VCenterScanConcurrencyLimit,
+		}, nil
+	}
+
+	if vjailbreakSettingsCM.Data["CHANGED_BLOCKS_COPY_ITERATION_THRESHOLD"] == "" {
+		vjailbreakSettingsCM.Data["CHANGED_BLOCKS_COPY_ITERATION_THRESHOLD"] = strconv.Itoa(constants.ChangedBlocksCopyIterationThreshold)
+	}
+
+	if vjailbreakSettingsCM.Data["VM_ACTIVE_WAIT_INTERVAL_SECONDS"] == "" {
+		vjailbreakSettingsCM.Data["VM_ACTIVE_WAIT_INTERVAL_SECONDS"] = strconv.Itoa(constants.VMActiveWaitIntervalSeconds)
+	}
+
+	if vjailbreakSettingsCM.Data["VM_ACTIVE_WAIT_RETRY_LIMIT"] == "" {
```

Comment:
if config changes , the updated config will not be available instantly without restart , can we implement hot reload ?